src/torrent.cpp

```diff
@@ -9227,22 +9227,31 @@ namespace libtorrent
 
 	void torrent::queue_up()
 	{
+		// fix async position change calls for race conditions on changed torrents
+		if(!m_auto_managed || m_abort || is_finished()) return;
```

Comment:
Jepp, 
You can't call up on finished torrent, it will run into the old set_queue_position assert